source/cfp.rst

```diff
@@ -0,0 +1,63 @@
+Call For Proposals
+==================
+
+We are planning to use `papercall.io`_ for handling CFP proposals.
+
+Daniele has set up a form with the following questions:
+
+Talk details
+------------
+
+Title
+  No help text
+Abstract
+  The abstract is the 300 character elevator pitch for this talk.
+Talk Format
+  What format is this talk best suited for?
+  Options: Talk; Workshop or tutorial; Code clinic, lunchtime event or other themed gathering; Poster
+Audience Level
+  Who is the best target audience for this talk?
+  Options: All; Beginner; Intermediate; Advanced
+Description
+  The description is the public abstract of your talk. The description will be seen by reviewers during the CFP process and will eventually be seen by the attendees of the event.
+  You should make the description of your talk as compelling and exciting as possible. Remember, you're selling both the organizers of the events to select your talk, as well as trying to convince attendees your talk is the one they should see.
+Notes
+  Notes will only be seen by reviewers during the CFP process. This is where you should explain things such as technical requirements, why you're the best person to speak on this subject, etc...
+Tags
+  Talk your talk to make it easier for event organizers to be able to find. Examples are "ruby, javascript, rails".
+
+
+Profile Details 
+----------------
+
+Name
+  No help text
+URL
+  Have a blog or some other website you'd like to have listed with this profile? If so, this is the field for you. 
+Organization or Affiliation
+  The name of your company, organization, or affiliation if you have any. 
+Twitter Handle
+  Twitter Handle should just be your handle without the "@". 
+Bio
+  Your bio sells you. Who are you? Why should people listen to you? Who's your favorite member of One Direction?
+  This is your chance to put it all out there. 
+
+
+Additional Information 
+-----------------------
+
+Please tell us a bit more about you. Feel free to mention any previous talks or workshops you’ve presented.
+
+
+Timing
+------
+
+If you are proposing a talk, workshop or gathering please indicate how long you would like to have for it.
+
+As a guide:
+
+- shorter talks: ~25 minutes
+- longer talks: ~45 minutes
+- workshops: typically 1-2 hours
+- lunchtime events: ~45 minutes
```

Comment:
Feedback from last year said it wasn't clear what the lunchtime events should be like; maybe we can clarify?